CMakeLists.txt

```diff
@@ -37,39 +37,39 @@ endif()
 #
 # If you change the version, please update the <version> tag in package.xml.
 
-project(dart)
+project(kido)
 
-set(DART_MAJOR_VERSION "5")
-set(DART_MINOR_VERSION "1")
-set(DART_PATCH_VERSION "1")
-set(DART_VERSION "${DART_MAJOR_VERSION}.${DART_MINOR_VERSION}.${DART_PATCH_VERSION}")
-set(DART_PKG_DESC "Dynamic Animation and Robotics Toolkit.")
-set(DART_PKG_EXTERNAL_DEPS "flann, ccd, fcl")
+set(KIDO_MAJOR_VERSION "0")
+set(KIDO_MINOR_VERSION "1")
+set(KIDO_PATCH_VERSION "0")
+set(KIDO_VERSION "${KIDO_MAJOR_VERSION}.${KIDO_MINOR_VERSION}.${KIDO_PATCH_VERSION}")
+set(KIDO_PKG_DESC "Dynamic Animation and Robotics Toolkit.")
```

Comment:
Sure, it should be changed to the full name of KIDO, which is "Kinematics, Dynamics and Optimization Library for Animation and Robotics" tentatively. This can be varied slightly. Thanks for the nice catch!